### PR TITLE
Conversation wrapper broken

### PR DESCRIPTION
**Pull request by stevenpkg**
the conversation date constant broke for me? also username and password fields commented out. Have fix and all works for me now and all test pass

### CODE REVIEW

**Comment by stevenpkg** [comment]
have also changed conversation plan back to free

**Comment by germanattanasio** [comment]
@stevenpkg is this still in place? I see you changed the version date to February and the workspace.json file.

**Comment by stevenpkg** [comment]
Hi German, yes thats correct, for me the constant ConversationV1.VERSION_DATE_2017_04_21 failed, so I put it back to a hard code date. The car workspace now has a slot example included

**Comment by germanattanasio** [comment]
Can you check why is failing? We should always use the latest version.

**Comment by stevenpkg** [comment]
Now use Conversation.VERSION_DATE_YYYY-MM-DD

File: app.js [hunk 1]
```diff
@@ -30,10 +30,9 @@ app.use(bodyParser.json());
 var conversation = new Conversation({
   // If unspecified here, the CONVERSATION_USERNAME and CONVERSATION_PASSWORD env properties will be checked
   // After that, the SDK will fall back to the bluemix-provided VCAP_SERVICES environment property
-  // username: '<username>',
-  // password: '<password>',
-  // url: 'https://gateway.watsonplatform.net/conversation/api',
-  version_date: Conversation.VERSION_DATE_2017_04_21
+  //'username': process.env.CONVERSATION_USERNAME,
+  //'password': process.env.CONVERSATION_PASSWORD,
+  'version_date': Conversation.VERSION_DATE_2017_04_21
```
Comment (germanattanasio) [inline app.js]: no need to use `'`

File: manifest.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ applications:
   command: npm start
   path: .
   memory: 256M
+
```
Comment (germanattanasio) [inline manifest.yml]: no need for space here

**Comment by stevenpkg** [comment]
@germanattanasio please approve. New workspace changes are needed, and there still are changes from months ago

**Pull request merged**